ogcapi-features/README.md

```diff
@@ -157,10 +157,10 @@ The following Link relations must exist in the Item object returned from the `/c
 | **rel**  | **href**                                     | **Media Type**       | **From**            | **Description**                                     |
 | -------- | -------------------------------------------- | -------------------- | ------------------- | --------------------------------------------------- |
 | `root`   | `/`                                          | application/json     | STAC API - Features | The root URI                                        |
-| `parent` | `/collections/{collectionId}`                | application/json     | OAFeat              | Parent reference, usually the containing Collection |
+| `collection` | `/collections/{collectionId}`            | application/json     | OAFeat              | The containing Collection |
```

Comment:
It's not "instead of" -- they're frequently the same (and always the same in all the STAC API impls I know of), but they don't have to be the same -- collection will always point to the Collection resource, but parent may *not* point to the collection, but might point to a parent Catalog.

I think we need to add to the definitions that there should be a `collection` link rel (as defined by STAC) in addition to `parent` (defined by STAC API)

This should also be added to the similar table in Item Search.

Reply:
> I think we need to add to the definitions that there should be a collection link rel (as defined by STAC) in addition to parent (defined by STAC API)

I restored the `parent` link in the latest commit, so the table now includes both `parent` and `collection` links.  Both use the same `href`.  Let me know if you think the `parent` link should use a different example `href`.

Reply:
> This should also be added to the similar table in Item Search.

I'm not finding this table.  Let me know if you've got a link.

Reply:
👍🏻